JWPlayer-SDK-iOS-Demo/HelperExtensions.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  HelperExtensions.swift
+//  JWPlayer-SDK-iOS-Demo
+//
+//  Created by Amitai Blickstein on 3/26/19.
+//  Copyright © 2019 JWPlayer. All rights reserved.
+//
+
+import Foundation
+
+extension JWPlayerController {
+    /// Autolayout support was introduced in SDK v3.3.0
+    @objc static var supportsAutolayout: Bool {
```

Comment:
@haawa799 please provide @AmitaiB  with feedback on how to best add the views

Reply:
I used the autolayout pattern in {{JWFreewheelVideoBaseView.m}}.